tests/test_coreg/test_affine.py

```diff
@@ -227,7 +229,8 @@ def test_coreg_translations__synthetic(self, fit_args, shifts, coreg_method) ->
         to be the right one; and that there is no other errors introduced in the process).
         """
 
-        # Initiate coregistration
+        warnings.filterwarnings("ignore", message="Covariance of the parameters*")
```

Comment:
I'm pretty sure this should be done within a context manager. But please correct me if there's a source that says it's not needed. The problem is that this setting might propagate to other subsequent tests.

```python
with warnings.catch_warnings():
    warnings.filterwarnings("ignore", message="Covariance of the parameters*")
    # here goes the warning-triggering code

```

Reply:
Yes, you have to do the context manager to target a specific bit of code. Here it applies to the entire test function. Because this function only has one call that can raise this type of warning, it's equivalent. And as the best practice to test something new is to write another small test function (something we didn't do so much at the beginning), it shouldn't become a problem! :smile: